### DISCUSSION TITLE
VtM Request Feature: Automated use of Blood and Willpower in Diciplines

### DISCUSSION THREAD

**Discussion by metehansakoglu** [JohanFalt/Foundry_WoD20#866] [Q&A] [answered]
Hello. First of all I want to thank you for this System Module. Me and my 5 players playing on this module nearly over a year. A little improvement came to my mind and I wish to share it with you. When Actors using Diciplines I belive reduce of Willpower and Blood Point can be automated. I know it`s a bit of little request but I belive in a long term it help a lot of people. 

A little not: *(I just want to help. If you`re thinking about the adding these feature you may want to add a little error box if Actor`s Blood and Willpower goes to zero)*

**Comment by JohanFalt** [comment, marked as answer]
I'll create an issue of this and see what I can do with it (and how).
